ex1_student_solution.py

```diff
@@ -337,31 +337,31 @@ def find_panorama_shape(src_image: np.ndarray,
         src_rows_num, src_cols_num, _ = src_image.shape
         dst_rows_num, dst_cols_num, _ = dst_image.shape
         src_edges = {}
-        src_edges['upper left corner'] = np.array([1, 1, 1])
-        src_edges['upper right corner'] = np.array([src_cols_num, 1, 1])
-        src_edges['lower left corner'] = np.array([1, src_rows_num, 1])
+        src_edges['upper left corner'] = np.array([0, 0, 1])
```

Comment:
Isnt this supposed to be the lower left corner? Or is it just a matter of semantics where you put your 0 axis?

Reply:
I think the reason they went with this notation for the corners is numpy notation for coordinates, where the origin of axes is on the top-left and the axes go down and to the right to match rows and columns respectively